papa/src/main/java/papa/InputTracker.kt

```diff
@@ -4,6 +4,9 @@ import android.view.InputEvent
 import android.view.KeyEvent
 import android.view.MotionEvent
 import papa.internal.RealInputTracker
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.DurationUnit.MILLISECONDS
```

Comment:
@0legg Should PY be using 310 alternatives here?

Reply:
I'm really taking a liking to Kotlin's duration class, looks nicer than the JDK ones.